CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Why add FATAL_ERROR which is deprecated ?

Reply:
Dropped.